### ISSUE TITLE
Use iot Adapter instead of internal server

### ISSUE THREAD

**Issue by ScumbagSteve** [ioBroker/ioBroker.geofency#99]
I did not want to open/expose my ioBroker server to the internet, therefore I was not able to use the adapter with it's internal server.
I use the iot Adapter, to have a secure interface via the internet.

[This ](https://jb-smarthome.de/iobroker-anwesenheitserkennung-mit-geofency) post describes a way to connect geofency via iot. Would be great if geofency could just use a free-to-choose datapoint as json input. Connection via iot would be rather simple I guess and would push the standard ioBroker adapters further.

**Comment by Apollon77** [comment]
Yes this is planned. We enhanced iot short time ago for netatmo with needed features
